config/runtime/server_internal_test.go

```diff
@@ -240,24 +249,6 @@ func TestServer_GetCORS(t *testing.T) {
 	}
 }
 
-func TestServer_ParseDuration(t *testing.T) {
```

Comment:
why this test is removed?

Reply:
Since parsing backend durations will happen on-the-fly with defaults this unit test is obsolete here.